node/src/RedisClusterClient.ts

```diff
@@ -388,6 +389,31 @@ export class RedisClusterClient extends BaseClient {
         );
     }
 
+    /** Echoes the provided `message` back.
+     * See https://redis.io/commands/echo for more details.
+     *
+     * @param message - The message to be echoed back.
+     * @param route - The command will be routed to a random node, unless `route` is provided, in which
+     *   case the client will route the command to the nodes defined by `route`.
+     * @returns The provided `message`. When specifying a route other than a single node,
+     *  it returns a dictionary where each address is the key and its corresponding node response is the value.
+     *
+     * @example
+     *      await client.echo("Glide-for-Redis");
+     *      "Glide-for-Redis"
+     *      await client.echo("Glide-for-Redis", "allNodes");
+     *      "Glide-for-Redis"
```

Comment:
shouldn't it return a dict?...

Reply:
If it returns a single response, you should fix documentation:
>      *  it returns a dictionary where each address is the key and its corresponding node response is the value.

Reply:
youre right, my bad